apps/user-office-backend/src/datasources/SEPDataSource.ts

```diff
@@ -62,6 +63,10 @@ export interface SEPDataSource {
     proposalPk: number
   ): Promise<boolean>;
   getSEPProposals(sepId: number, callId: number | null): Promise<SEPProposal[]>;
+  getSEPUsersByProposalPkAndCallId(

```

Comment:
Nitpick: I think this could have been part of another PR where you add specific actions after adding an engine. But it's fine :+1: